### ISSUE TITLE
spanning cell issue

### ISSUE THREAD

**Issue by tzktok** [whn09/table_structure_recognition#9]
I need to  annotate dataset that contains span cell..I have a doubt on span cell annotation..
below is my table image..

![Screenshot (19)](https://github.com/user-attachments/assets/1dc3fae7-7041-41f6-8084-54cd56ff0266)

is annotated span cells are correct ?

![Screenshot (20)](https://github.com/user-attachments/assets/2808c223-11b0-467d-876b-89f1fa9c2bea)

pls..clarify this @whn09 ..thx in advance!!!

**Comment by whn09** [comment]
You can refer to this image:

![139559159-cd23c972-8731-48ed-91df-f3f27e9f4d79](https://github.com/user-attachments/assets/c2561b39-a006-4a36-8aef-7e09b02f0c6a)

**Comment by tzktok** [comment]
i am bit confused in  spanningcell image...could u draw  the spanningcell for  below image @whn09 ...thx in advance!!

![bank_of_baroda pdf_page_91](https://github.com/user-attachments/assets/7b5763c2-f8e8-4aee-964f-672a466681f3)

**Comment by whn09** [comment]
I think only "Page Total:" is a spanning cell.

**Comment by tzktok** [comment]
> I think only "Page Total:" is a spanning cell.
 
the content in particulars have been span upto  deposit column  see the above image @whn09 ...so i confused where to annotate spanning cell for that ?

**Comment by whn09** [comment]
I don't think you should label the UPI/XXXX as spanning cells, and you should treat them as normal columns and rows.
